server/Episememe.Application/Features/UpdateTags/UpdateTagsCommand.cs

```diff
@@ -0,0 +1,26 @@
+using MediatR;
+using System;
+using System.Collections.Generic;
+
+namespace Episememe.Application.Features.UpdateTags
+{
+    public class UpdateTagsCommand : IRequest
+    {
+        public string Id { get; }
+        public IEnumerable<string> Tags { get; }
+    
+
+        private UpdateTagsCommand(string id, IEnumerable<string> tags)
+        {
+            Id = id;
+            Tags = tags;
+        }
+
+        public static UpdateTagsCommand Create(string id, IEnumerable<string> tags)
+        {
+            if (id == null || tags == null)
```

Comment:
Throw exceptions with a message, at least a name of the argument, like that: `new ArgumentNullException(nameof(tags))`